hokusai/development.yml

```diff
@@ -2,24 +2,39 @@
 version: '2'
 services:
   rosalind:
-    command: sh -c 'bundle exec rails db:setup && bundle exec puma -C config/puma.config'
+    command: sh -c 'bundle exec rails db:migrate && bundle exec puma -C config/puma.config'
     env_file: ../.env
     extends:
       file: build.yml
       service: rosalind
     ports:
-      - 3000:3000
+      - 5000:5000
     volumes:
       - ../:/app
     depends_on:
       - rosalind-postgres
+      - rosalind-redis
     environment:
       - DATABASE_URL=postgresql://postgres:@rosalind-postgres/rosalind_development
       - RAILS_ENV=development
       - REDIS_URL=redis://rosalind-redis
+  worker:
+    extends:
+      file: build.yml
+      service: rosalind
+    environment:
+      - RAILS_ENV=development
+      - DATABASE_URL=postgresql://postgres:@rosalind-postgres/rosalind_development
+      - REDIS_URL=redis://rosalind-redis
+    env_file: ../.env
+    command: bundle exec sidekiq
+    depends_on:
+      - rosalind-postgres
+      - rosalind-redis
   rosalind-postgres:
-    image: postgres:10.6
+    image: postgres:9.6
```

Comment:
This was also key — previously I had 10.6 to match the current Heroku environment.